### DISCUSSION TITLE
Copilot

### DISCUSSION THREAD

**Discussion by Dex-Lab27** [community/community#62767] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

how?

**Comment by Spixmacaw** [comment]
GitHub Copilot is an AI pair programmer that offers autocomplete-style suggestions as you code. You can receive suggestions from GitHub Copilot either by starting to write the code you want to use, or by writing a natural language comment describing what you want the code to do.

**Comment by DSDmark** [comment]
Hi @Dex-Lab27,

Can you **specific more** about the details of what you want? For better understanding 😅. Also, you can add some **screenshots** or **error** which you were facing 👍.
